### PR TITLE
docs: add debug unit test command into package.json

### PR DESCRIPTION
**Pull request by hanwencheng**
And update Docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,18 @@ Corresponding data:
 }
 ```
 
+#### Unit Test
+
+Run `yarn test` for all the units test.
+
+If debugging is needed:
+
+1. Insert a line of `debugger;` into the code base where might be failed.
```
Comment (Tbaut) [inline README.md]:
```suggestion
1. Insert `debugger;` in the code where you think it fails.
```

File: README.md [hunk 2]
```diff
@@ -117,6 +117,18 @@ Corresponding data:
...
+2. Run `yarn test:debug`
+3. Open a new tab in Chrome with address `chrome://inspect`
```
Comment (Tbaut) [inline README.md]:
```suggestion
3. Open a new tab in Chrome and go to `chrome://inspect`
```

File: README.md [hunk 3]
```diff
@@ -117,6 +117,18 @@ Corresponding data:
...
+4. Click the `inspect` button of target under `Remote Target`
+5. Back to the terminal and choose one of the node watch commands for running tests again.
```
Comment (Tbaut) [inline README.md]:
```suggestion
5. Back to the terminal, choose one of the node watch commands to run the tests again.
```

**Comment by hanwencheng** [comment]
Thanks for the rephrasing, updated.

**Pull request merged**